### ISSUE TITLE
Integrate linting support and compliance with flake8

### ISSUE THREAD

**Issue by jrgriffiniii** [pulibrary/dspace-python#3]
This currently triggers a substantial number of failures:

```
./thesiscentral-vireo/dataspace/python/vireoSpreadSheetSplit.py:13:1: E302 expected 2 blank lines, found 1
./thesiscentral-vireo/dataspace/python/vireoSpreadSheetSplit.py:16:41: E251 unexpected spaces around keyword / parameter equals
./thesiscentral-vireo/dataspace/python/vireoSpreadSheetSplit.py:16:43: E251 unexpected spaces around keyword / parameter equals
./thesiscentral-vireo/dataspace/python/vireoSpreadSheetSplit.py:17:5: F841 local variable 'sheet' is assigned to but never used
./thesiscentral-vireo/dataspace/python/vireoSpreadSheetSplit.py:29:63: W291 trailing whitespace
./thesiscentral-vireo/dataspace/python/vireoSpreadSheetSplit.py:30:80: E501 line too long (88 > 79 characters)
./thesiscentral-vireo/dataspace/python/vireoSpreadSheetSplit.py:30:89: W291 trailing whitespace
./thesiscentral-vireo/dataspace/python/vireoSpreadSheetSplit.py:32:80: E501 line too long (88 > 79 characters)
./thesiscentral-vireo/dataspace/python/vireoSpreadSheetSplit.py:33:80: E501 line too long (106 > 79 characters)
./thesiscentral-vireo/dataspace/python/vireoSpreadSheetSplit.py:37:80: E501 line too long (98 > 79 characters)
[...]
./ezids/ezid.py:233:3: E111 indentation is not a multiple of four
./ezids/ezid.py:234:3: E111 indentation is not a multiple of four
./ezids/ezid.py:237:23: E701 multiple statements on one line (colon)
./ezids/ezid.py:242:43: E502 the backslash is redundant between brackets
./ezids/ezid.py:243:3: E128 continuation line under-indented for visual indent
./ezids/ezid.py:244:32: E721 do not compare types, use 'isinstance()'
./ezids/ezid.py:244:55: E502 the backslash is redundant between brackets
./ezids/ezid.py:245:3: E128 continuation line under-indented for visual indent
./ezids/ezid.py:246:3: E111 indentation is not a multiple of four
./ezids/ezid.py:251:3: E111 indentation is not a multiple of four
./ezids/ezid.py:252:3: E111 indentation is not a multiple of four
./ezids/ezid.py:254:3: E111 indentation is not a multiple of four
./ezids/ezid.py:256:3: E111 indentation is not a multiple of four
./ezids/ezid.py:257:3: E111 indentation is not a multiple of four
./ezids/ezid.py:259:3: E111 indentation is not a multiple of four
./ezids/ezid.py:260:3: E111 indentation is not a multiple of four
./ezids/ezid.py:262:3: E111 indentation is not a multiple of four
./ezids/ezid.py:264:3: E111 indentation is not a multiple of four
./ezids/ezid.py:265:3: E111 indentation is not a multiple of four
./ezids/ezid.py:267:3: E111 indentation is not a multiple of four
./ezids/ezid.py:268:3: E111 indentation is not a multiple of four
./ezids/ezid.py:269:3: E111 indentation is not a multiple of four
./ezids/ezid.py:271:3: E111 indentation is not a multiple of four
./ezids/ezid.py:272:3: E111 indentation is not a multiple of four
./ezids/ezid.py:274:3: E111 indentation is not a multiple of four
./ezids/ezid.py:276:3: E111 indentation is not a multiple of four
./ezids/ezid.py:277:3: E111 indentation is not a multiple of four
./ezids/ezid.py:279:3: E111 indentation is not a multiple of four
./ezids/ezid.py:280:3: E111 indentation is not a multiple of four
./ezids/ezid.py:281:3: E111 indentation is not a multiple of four
./ezids/ezid.py:283:3: E111 indentation is not a multiple of four
./ezids/ezid.py:284:3: E111 indentation is not a multiple of four
./ezids/ezid.py:285:5: E127 continuation line over-indented for visual indent
./ezids/ezid.py:286:3: E111 indentation is not a multiple of four
./ezids/ezid.py:288:3: E111 indentation is not a multiple of four
./ezids/ezid.py:289:3: E111 indentation is not a multiple of four
./ezids/ezid.py:291:3: E111 indentation is not a multiple of four
./ezids/ezid.py:293:3: E111 indentation is not a multiple of four
./ezids/ezid.py:295:3: E111 indentation is not a multiple of four
./ezids/ezid.py:296:3: E111 indentation is not a multiple of four
```

**Comment by kmcelwee** [comment]
Blocked by Issue #4 and will be closed by PR #18

**Comment by kmcelwee** [comment]
Closed by PR #18
